ITRStartTask/AppLibrary/ViewControllersFactory.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Fabric.swift
+//  ITRStartTask
+//
+//  Created by Homyakov, Ilya2 on 20/08/2018.
+//  Copyright © 2018 Homyakov, Ilya2. All rights reserved.
+//
+
+import UIKit
+
+class ViewControllersFactory {
+    static func getStoriesViewController () -> UIViewController {
+        let view = StoriesViewController(nibName: "StoriesViewController", bundle: nil)
```

Comment:
Add extension for UIViewController for XIB instance

Reply:
Fixed